samples/04_gis_analysts_data_scientists/Training a Deep Learning model, for using it in 3D basemap Solutions..ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<h1>Training a Deep Learning model for Point Cloud Classification</h1><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/763/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
'corrected accordingly'', Thanks.